src/fhirdefs/FHIRDefinitions.ts

```diff
@@ -24,6 +25,11 @@ export class FHIRDefinitions implements Fishable {
     this.codeSystems = new Map();
     this.implementationGuides = new Map();
     this.predefinedResources = new Map();
+
+    // FHIR R4 does not have a StructureDefinition that defines "Base" but FHIR R5 does.
+    // We have defined a "placeholder" StructureDefinition for "Base" for R4.
+    // Inject the R4 "Base" placeholder StructureDefinition
+    this.add(JSON.stringify(STRUCTURE_DEFINITION_R4_BASE));
```

Comment:
I couldn't figure out why the order was still causing problems for me when I did manual testing using R5 (but somehow passed the R4-based unit tests).  It turns out that this line is the culprit.  The argument to `add` should be a JSON _object_, not a string.  When you do it like this (as a string), then `Base` doesn't get registered in the definitions at all (for R4) -- so lookups for `Base` still return `undefined` in R4.

Once you fix this, it cause's the tanks `getSdType` function to operate a little differently -- which, in turn, causes an error.  We _do_ want to be able to fish for `Base` in R4, so I think you need to fix this here (by removing the `stringify`) and then also make a fix in `getSdType` to get the now-breaking test passing again.